attestation-evidence-broker/docs/design/design.md

```diff
@@ -0,0 +1,113 @@
+This document provides that how to use Attestation Evidence Broker (AEB) with the EAA KBC module in Attestation Agent to support **SEV (-ES) runtime remote attestation**.
+
+# Background of SEV (-ES) runtime remote attestation
+
+Compared with Intel TDX and SEV-SNP, the runtime remote attestation of SEV(-ES) does NOT support dynamic measurement and can only verify static measurement values, which are always constant values [1]. Although SEV(-ES) has this limitation, runtime remote attestation is still necessary. When the SEV guest is running, system can report trusted heartbeat health information to the user by sending attestation evidence.
```

Comment:
s/necessary/useful/
s/system can report trusted heartbeat health information to the user by sending attestation evidence./it can report the attestation evidence with fixed launch digest as a heartbeat for trusted healthy check./